crates/utils/tests/serde.js

```diff
@@ -0,0 +1,20 @@
+const wasm = require('wasm-bindgen-test.js');
+const assert = require('assert');
```

Comment:
Only `deepStrictEquals` is used from this module, which can be implemented as

```js
function deepStrictEquals(left, right) {
    if (left !== right) {
        throw Error(`${left} != ${right}`}
    }
}
```
That way, we can run these tests in browser too

Reply:
oh, thanks. I will change it right now.

Reply:
Turns you it's a little more involved than that, but this use case I have a work around.